astropy/visualization/wcsaxes/tests/test_wcsapi.py

```diff
@@ -460,7 +460,7 @@ def test_coord_meta_wcsapi():
     assert coord_meta['default_axis_label'] == ['Frequency', 'time', 'RA', 'DEC', 'phys.polarization.stokes']
 
 
-@pytest.mark.remote_data(source='astropy')
+@pytest.mark.remote_data
```

Comment:
This is the other useful thing about having a wrapper decorator, you can make one decorator do remote data and mpl_image_compare ;)

Reply:
Pull requests welcome! :wink: